src/operations/PatchOperation.php

```diff
@@ -0,0 +1,10 @@
+<?php declare(strict_types=1);
+
+namespace blancks\JsonPatch\operations;
+
+abstract class PatchOperation
```

Comment:
The intent is these DTOs will generally only have properties, not methods, and they will always require at least an `op` property. 

Therefore I didn't think there was any benefit in also defining an (empty) interface: users who wish to define a custom operation object should always be able to extend the abstract `PatchOperation` class.